Dockerfile.dapper

```diff
@@ -1,25 +1,20 @@
-FROM golang:1.16.9-buster
+FROM registry.suse.com/bci/golang:1.17
 
-ENV ARCH amd64
+ARG DAPPER_HOST_ARCH="amd64"
+ENV ARCH=${DAPPER_HOST_ARCH}
 
 ARG http_proxy=$http_proxy
 ARG https_proxy=$https_proxy
 ARG no_proxy=$no_proxy
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates
+RUN zypper -n in go1.17 curl bash git jq
```

Comment:
For the BCI Golang image is not necessary to specify `go1.17`, as it's already in the image.